unlock-app/src/selectors/currency.js

```diff
@@ -31,9 +18,10 @@ export function formatEth(eth) {
  */
 export function formatCurrency(amount) {
   let currency = Number(amount)
-  if (currency < 1 && currency > 0.01)
+  if (currency === 0) return '0'
+  if (currency < MINIMUM_THRESHOLD) return '< 0.0001'
+  if (currency < 1)
     return parseFloat(currency.toPrecision(SIGNIFICANT_DIGITS)).toString()
-  if (currency < 0.01) return '0'
   if (currency >= 1000 && currency < 1e5)
     return Math.round(currency).toLocaleString()
```

Comment:
I'm curious to know if the usage of `Number.toLocaleString()` alongside the hardcoded test results could cause problems in the future if we get European contributors, for example (`,` vs `.`). Not something to be addressed now, but a possible point of friction.